app/konnect/dev-portal/index.md

```diff
@@ -5,24 +5,17 @@ content_type: explanation
 
-## Publish service documentation
+## Dev Portal use cases
 
-In {{site.konnect_short_name}}, API products can be published to the Dev Portal with one click. Published API products become immediately available to users who have access to the Dev Portal. When an API product is published, the API spec and any product documentation becomes discoverable. We use the term discoverable here because the Dev Portal can create a unified API experience where a developer can navigate through the different APIs that are available, read documentation, test endpoints within the Dev Portal, and register to create applications for specific APIs. 
+You can use the following table to help you determine which Dev Portal configurations are best for your use case:
 
-* [Getting started guide](/konnect/getting-started/publish-service/): This guide walks you through importing an API spec, publishing your service and corresponding API spec to the Dev Portal, and configuring registration settings so that users can view your services and register applications.
-
-* [API product documentation](/konnect/api-products/service-documentation/): This doc explains how to upload, edit, and publish product documentation using the **API Products** dashboard and publish API product to the Dev Portal to be consumed by your users.
-
-* [Manage Konnect API product versions](/konnect/api-products/): This explains how to manage the API product version for your services, including the status of the API product version. API product versions can have a status of "Published", "Deprecated", or "Unpublished". 
-
-### API specification
-
-API specifications can be uploaded and attached to a specific API product version within the [**API Product**](https://cloud.konghq.com/api-products/). 
-You can have different API specs that correspond to different API product versions.
-{{site.konnect_short_name}} accepts OpenAPI (Swagger) specs in YAML or JSON.
-
-Once you've uploaded the spec, you can also preview the way the spec will render, including the methods available, endpoint descriptions, and example values. You'll also be able to filter by tag when in full-page view. 
-The Dev Portal lives at a separate URL from {{site.konnect_short_name}} and requires that all users, including Admin roles, [register as a Developer](/konnect/dev-portal/dev-reg/).
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Publish APIs to two Dev Portals; one for internal developers and one for external partner developers | [Multi-portal](/konnect/dev-portal/create-dev-portal/) |
+| Let developers get API keys and start using your APIs | [Enable app reg and Dev Portal](/konnect/dev-portal/applications/enable-app-reg/) |
+| Segment which users can see which APIs | [Assign different APIs and permissions with RBAC Teams](/konnect/api/portal-auth/portal-rbac-guide/#main) |
```

Comment:
added comments in a pr elsewhere...struggling to find it but I think we should revert this page to how it was before